.github/workflows/source-web-tool-scripts.yaml

```diff
@@ -1,32 +1,15 @@
-on:
-  push:
-    branches:
-      - main
-      - master
-  pull_request:
-    branches:
-      - main
-      - master
-
-name: Source web-tool scripts
-
+name: Source web tool scripts in container r2dii/r-packages
+on: push
 jobs:
-  Test-r-scripts:
-    runs-on: ${{ matrix.config.os }}
-
-    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
-
+  Source-web-tool-scripts:
+    container: 2dii/r-packages:latest
```

Comment:
This is the most interesting bit.

Reply:
is this pulling from Docker Hub or from GH Containers? just curious

Reply:
I didn't setup GH Containers, so I think it's pulling from Docker Hub.